### ISSUE TITLE
Team names disappear when scrolling the Rankings views

### ISSUE THREAD

**Issue by IAmAGitNewbie** [the-blue-alliance/the-blue-alliance-android#356]
I was using the latest app to check out the rankings at the Arizona East regional while it is in progress.  The list displays fine to begin with:

![screenshot_01](https://cloud.githubusercontent.com/assets/2994465/6759005/dbe1661e-cef9-11e4-8b81-601a3ef4a384.png)

but after some scrolling either up or down I start to see team names not render.  The rest of their info renders fine, just not their names:

![screenshot_03](https://cloud.githubusercontent.com/assets/2994465/6759014/f51fee16-cef9-11e4-8b22-431ea3948b60.png)

The more I scroll the list up/down, the more team names will disappear and not reappear.  

Workaround: Swap over to another view like Matches and back.

**Comment by phil-lopreiato** [comment]
Thanks for the report!

This is strange. I can't reproduce on my device. 

1) What version of android do you have?
2) What's your app version (go settings, and what version number do you see)

**Comment by IAmAGitNewbie** [comment]
1: Samsung Galaxy S3 running 4.4.2
2: App version 2.1.12

**Comment by phil-lopreiato** [comment]
Thanks - I'll take a look.

@nwalters512 any ideas? Seems like an issue with the ViewHolder?

**Comment by IAmAGitNewbie** [comment]
2 quick updates on what I have been able to observe:

1:  I got a score notification for a team at another event so I looked at it and then used the back button to return to the rankings view.  For about 1.5 seconds the view was all blank and then it filled in with all of the names that had disappeared.  The ranking list was not in motion and I was not touching the screen after the pressing Back.

2: I observe a slight scrolling lag whenever names disappear from the list as I scroll.  When I scroll and there is no lag, no names disappear.  Occasionally there is a very slight pause (~.5 second) and thats when one or more team names disappear.

**Comment by nwalters512** [comment]
I have no idea for this... I'm tempted to chalk it up to Samsung being stupid, but we would've heard from our other S3 users if this was widespread. OP, do you know how to check LogCat to see if anything is being printed when this happens?

**Comment by IAmAGitNewbie** [comment]
I captured 5 different logcats and tried to pare them down in case you want to see the full logs.  For the last one I FC'd the app and started it from the Launcher.   I immediately went into the event and then to the rankings.  Once it displayed I repeated the test.  When the problem reproduced I stopped and grabbed the latest tba related entries. 

I found this:

I/tba-android:refresh(28086): Event 2015azch Rankings refresh complete
I/InputReader(  790): Touch event's action is 0x0 (deviceType=0) [pCnt=1, s=0.5819 ] when=35903966137000
I/InputDispatcher(  790): Delivering touch to: action: 0x4, toolType: 1
I/InputDispatcher(  790): Delivering touch to: action: 0x0, toolType: 1
I/ServiceKeeper(  790): In getseinfo pid = 790 uid = 1000 seinfo= system
D/dalvikvm(28086): GC_CONCURRENT freed 4394K, 37% free 20316K/32208K, paused 8ms+18ms, total 133ms
D/dalvikvm(28086): WAIT_FOR_CONCURRENT_GC blocked 69ms
I/tba-android:refresh(28086): Event 2015azch Results refresh complete
I/InputReader(  790): Touch event's action is 0x1 (deviceType=0) [pCnt=1, s=] when=35904190950000
I/InputDispatcher(  790): Delivering touch to: action: 0x1, toolType: 1
E/SMD     (  173): DCD ON
I/InputReader(  790): Touch event's action is 0x0 (deviceType=0) [pCnt=1, s=0.5820 ] when=35904899968000
D/InputDispatcher(  790): Waiting for application to become ready for input: xxxx Reason: Waiting because the touched window has not finished processing the input events that were previously delivered to it.
I/ServiceKeeper(  790): In getseinfo pid = 790 uid = 1000 seinfo= system
I/InputDispatcher(  790): Delivering touch to: action: 0x4, toolType: 1
I/InputDispatcher(  790): Delivering touch to: action: 0x0, toolType: 1
I/tba-android:refresh(28086): Event 2015azch Awards refresh complete
I/tba-android:refresh(28086): Event 2015azch Rankings refresh complete

If you want to see the full ~1600 line long from the launch to failure I can provide that.
